src/screens/App/screens/Dashboard/index.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import DashboardScreen from './screen';
```

Comment:
We are naming the presentational components Layout.

Reply:
Changed in #2